src/libraries/System.Net.Http/tests/StressTests/HttpStress/StressServer.cs

```diff
@@ -315,6 +317,13 @@ private static void MapRoutes(IEndpointRouteBuilder endpoints)
             });
         }
 
+        private static void WorkaroundAssemblyResolutionIssues()
+        {
+            // For some reason, System.Security.Cryptography.Encoding.dll fails to resolve when being loaded on-demand by AspNetCore.
+            // Enforce early-loading to workaround this issue.
+            _ = new Oid();
```

Comment:
Here is the error I get by removing this line:
https://dev.azure.com/dnceng/public/_build/results?buildId=1459792&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=6c56d998-158d-5232-283c-5140104799fc

Reply:
@wfurt any ideas? Should we file this as a separate issue? Or is this related to all the targeting pack magic around?